tests/unit/workflows/test_pipeline.py

```diff
@@ -233,6 +234,26 @@ def test_init(args):
     assert isinstance(workflow.dpath_pipeline_bids_db, Path)
 
 
+@pytest.mark.parametrize(
+    "args, exception_class, exception_message",
+    [
+        (
+            {
+                "dpath_root": "my_dataset",
+                "pipeline_name": "my_pipeline",
+                "hpc": "slurm",
+                "write_list": "list.tsv",
+            },
+            ValueError,
+            "HPC job submission and writing a list of participants and sessions are mutually exclusive.",  # noqa: E501
+        ),
+    ],
+)
```

Comment:
It might be cleaner to have the values directly in the test, since we only have a single set of value tested.